### DISCUSSION TITLE
关于播放器置顶功能的问题

### DISCUSSION THREAD

**Discussion by QingW-7023** [the1812/Bilibili-Evolved#2825] [功能建议 / Ideas]
开启“默认宽屏”，再开启“播放器置顶”，打开新播放页面。
页面加载成功并且自动切换到宽屏后，视频有一半在可见屏幕之外（上面），需鼠标向上滚动才能看见整个完整的播放窗口。
![image](https://user-images.githubusercontent.com/64522280/147979333-21cc9a5f-6b76-40a6-b7e2-18a69c1c2856.png)

**Comment by Geeyun-JY3** [comment]
老问题，请先善用搜索：#483

**Reply by QingW-7023** [reply]
噢不好意思，这个页面之前有看但没懂，现在仔细一看竟是同一个问题。
